open-sphere-base/core/src/main/java/io/opensphere/core/pipeline/processor/PointSpriteProcessor.java

```diff
@@ -262,7 +262,7 @@ protected void processTextureLoaded(Collection<? extends PointSpriteGeometry> te
         readLock.lock();
         try
         {
-            for (PointSpriteGeometry geom : textureLoaded)
+            textureLoaded.parallelStream().forEach(geom ->

```

Comment:
how large can pointSpriteGeometry textureLoaded get? Is it worth it to create a parallelStream ?